<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NMES V0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Connect">
         <attribute name="title">
          <string>Connect</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBoxWirelessInterface">
            <property name="title">
             <string>Wireless interface:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButtonBTdiscoverDevices">
                 <property name="text">
                  <string>Discover BT</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QRadioButton" name="radioButtonWiFi">
                 <property name="text">
                  <string>WiFi</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="labelBTstatus">
                 <property name="text">
                  <string>Status:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioButtonBT">
                 <property name="text">
                  <string>Bluetooth</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QComboBox" name="comboBoxBTdevices"/>
               </item>
               <item row="5" column="0">
                <widget class="QPushButton" name="pushButtonBTconnect">
                 <property name="text">
                  <string>Connect BT</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>IP Address:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditWiFiAddress">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="5" column="2">
                <widget class="QPushButton" name="pushButtonWiFiConnect">
                 <property name="text">
                  <string>Connect WiFi</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLabel" name="labelWiFiStatus">
                 <property name="text">
                  <string>Status:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="labelWiFiPort">
                   <property name="text">
                    <string>Port:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditWiFiPort">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="6" column="2">
                <widget class="QPushButton" name="pushButtonWiFiDisconnect">
                 <property name="text">
                  <string>Disconnect WiFi</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QPushButton" name="pushButtonBTdisconnect">
                 <property name="text">
                  <string>Disconnect BT</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Config</string>
         </attribute>
         <widget class="QLineEdit" name="lineEditAmplitude">
          <property name="geometry">
           <rect>
            <x>144</x>
            <y>10</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonAmplitude">
          <property name="geometry">
           <rect>
            <x>292</x>
            <y>10</y>
            <width>133</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Amplitude</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>128</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Amplitude 0-31mA</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditFrequency">
          <property name="geometry">
           <rect>
            <x>108</x>
            <y>47</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonFrequency">
          <property name="geometry">
           <rect>
            <x>256</x>
            <y>47</y>
            <width>134</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Frequency</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>47</y>
            <width>92</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Frequency Hz</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>83</y>
            <width>156</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Phase Duration (us/50)</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditPhase">
          <property name="geometry">
           <rect>
            <x>172</x>
            <y>83</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonPhase">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>83</y>
            <width>141</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Pulse width</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditSymetry">
          <property name="geometry">
           <rect>
            <x>132</x>
            <y>120</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>116</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Symmetry Factor</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonSymetry">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>120</y>
            <width>132</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Symmetry</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditON">
          <property name="geometry">
           <rect>
            <x>91</x>
            <y>157</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>157</y>
            <width>75</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>On time (s)</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonOnTime">
          <property name="geometry">
           <rect>
            <x>239</x>
            <y>157</y>
            <width>119</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update ON time</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>194</y>
            <width>78</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Off time (s)</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditOFF">
          <property name="geometry">
           <rect>
            <x>94</x>
            <y>194</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonOFF">
          <property name="geometry">
           <rect>
            <x>242</x>
            <y>194</y>
            <width>124</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update OFF time</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonRampUp">
          <property name="geometry">
           <rect>
            <x>288</x>
            <y>230</y>
            <width>122</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Ramp up</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>124</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Ramp up time (ds)</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditRampUp">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>230</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonRampDown">
          <property name="geometry">
           <rect>
            <x>308</x>
            <y>267</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Ramp down</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>267</y>
            <width>144</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Ramp down time (ds)</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditRampDown">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>267</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditContractions">
          <property name="geometry">
           <rect>
            <x>162</x>
            <y>304</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>304</y>
            <width>146</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Number Contractions</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonContractions">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>304</y>
            <width>149</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Contractions</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonChannel1">
          <property name="geometry">
           <rect>
            <x>232</x>
            <y>341</y>
            <width>129</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Channel 1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>341</y>
            <width>68</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Channel 1</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditChannel1">
          <property name="geometry">
           <rect>
            <x>84</x>
            <y>341</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonChannel2">
          <property name="geometry">
           <rect>
            <x>232</x>
            <y>377</y>
            <width>129</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Channel 2</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditChannel2">
          <property name="geometry">
           <rect>
            <x>84</x>
            <y>377</y>
            <width>142</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>377</y>
            <width>68</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Channel 2</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>AT</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEditAT"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonATSend">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPlainTextEdit" name="plainTextEditAT"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Plot</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Measurement">
    <property name="title">
     <string>&amp;Measurement</string>
    </property>
    <addaction name="action_Run"/>
    <addaction name="action_Save"/>
    <addaction name="action_Clean"/>
   </widget>
   <addaction name="menu_Measurement"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Run"/>
   <addaction name="action_Save"/>
   <addaction name="action_Clean"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Run">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Button-Play-icon.png</normaloff>:/images/images/Button-Play-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Save-icon.png</normaloff>:/images/images/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Clean">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Clean-icon.png</normaloff>:/images/images/Clean-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clean</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonBTdiscoverDevices</tabstop>
  <tabstop>comboBoxBTdevices</tabstop>
  <tabstop>pushButtonBTconnect</tabstop>
  <tabstop>lineEditAT</tabstop>
  <tabstop>pushButtonATSend</tabstop>
  <tabstop>radioButtonBT</tabstop>
  <tabstop>lineEditWiFiPort</tabstop>
  <tabstop>pushButtonBTdisconnect</tabstop>
  <tabstop>pushButtonWiFiDisconnect</tabstop>
  <tabstop>radioButtonWiFi</tabstop>
  <tabstop>lineEditWiFiAddress</tabstop>
  <tabstop>plainTextEditAT</tabstop>
  <tabstop>pushButtonWiFiConnect</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
